include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -5266,6 +5272,13 @@ struct OrtApi {
    * \since Version 1.23
    */
   ORT_API2_STATUS(GetTensorSizeInBytes, _In_ const OrtValue* ort_value, _Out_ size_t* size);
+
+  /** \brief Calls OrtAllocator::GetStats function
+   *
+   * \since Version 1.23.
+   */
```

Comment:
Please, provide documentation with parameters description and what it returns when the allocator does not support statistics.